health-sdk/sdk/src/main/res/values/strings.xml

```diff
@@ -52,4 +52,6 @@
     <string name="ghs_more_information_fragment_title">Mehr Informationen</string>
     <string name="ghs_privacy_policy">Datenschutzerklärung</string>
     <string name="ghs_privacy_policy_link_url"/>
+
+    <string name="ghs_payment_provider_logo_content_description">logo</string>
```

Comment:
@a-szotyori , I added this so we can have a content description for the payment provider icons list. Since we don't currently have a large icon for the payment provider, it doesn't make sense to me to make the icons bigger when the font is increased. Perhaps when we will have larger icons we can increase the size of the logos as well.

Thoughts?

Reply:
I agree. The important thing is that text size increases. The size of the logos themselves we don't need to change ourselves when the font size changes. I quickly checked that Android will increase their size automatically when users increase the "display size" so we are fine leaving the resizing to Android:
![Screenshot_20240404_203812](https://github.com/gini/gini-mobile-android/assets/13149498/914119c4-5187-4de1-b133-d56e66335e45)

Reply:
Thanks for confirming! Then all that's left for this to be ready for review is the translation for `logo`.  I couldn't think of a more descriptive word, so any other suggestions are welcome.